README.md

```diff
@@ -53,6 +53,23 @@ Latest installers with PyPy 3.9 in the base environment:
 | OS X    | x86_64                | [Miniforge-pypy3-MacOSX-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge-pypy3-MacOSX-x86_64.sh) |
 | Windows | x86_64                | [Miniforge-pypy3-Windows-x86_64](https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge-pypy3-Windows-x86_64.exe) |
 
+<details><summary>Mambaforge (Discouraged as of September 2023)</summary>
+
+With the `release
+<https://github.com/conda-forge/miniforge/releases/tag/23.3.1-0>`__ of
+``Miniforge3-22.3.1-0``, that incorporated the changes in
+[#277](https://github.com/conda-forge/miniforge/pull/277), the packages and
+configuration of ``Mambaforge`` and ``Miniforge3`` are now **identical**. The
+only difference between the two is the name of the installer and, subsequently,
+the default installation directory.
+
+Given its wide usage, there are no plans to deprecate Mambaforge. If at some
+point we decide to deprecate Mambaforge, it will be appropriately announced and
+communicated with sufficient time in advance.
+
+As of September 2023, the new usage of Mambaforge is thus discouraged. Bug
+reports specific to Mambaforge will be closed as won't fix.
```

Comment:
I think this is the only sentence of value I am suggesting to add.

I really don't want to spend time discussing any issues specific to Mambaforge.

Reply:
This is the only sentence I don't understand. Since Mambaforge and Miniforge3 are basically identical, I don't understand this specific sentence.

Reply:
i feel like small bugs like this: https://github.com/conda-forge/miniforge/issues/399 
might keep coming up in the future, and I would like to respectfully close them reducing the complexity of our installers.

Reply:
Ah makes sense.

Reply:
Yep. Identical is one thing, what users may request is another. Also, if we ever decide to stop publishing mambaforge, we need to make sure folks understand it is deprecated. Nothing like "no issues accept" to convey that message ;-p